lighthouse-core/audits/cache-start-url.js

```diff
@@ -37,21 +37,18 @@ class CacheStartUrl extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    let cacheHasStartUrl = false;
-    const manifest = artifacts.Manifest.value;
-    const cacheContents = artifacts.CacheContents;
-
-    if (!(manifest && manifest.start_url && manifest.start_url.value)) {
+    if (!artifacts.Manifest || !artifacts.Manifest.value) {
```

Comment:
Hmm, maybe I don't understand what you're saying.

Just to be clear, by invalid JSON I mean just unparseable JSON; if the manifest is parseable by `JSON.parse` but is invalid according to the manifest spec, that *will* show up in the individual audits.

This code *can* separate these two states, but I put them in one conditional on purpose because we want the audits to just fail and not output a `debugString` (otherwise we'd have 12 `debugString`s or whatever in the report saying the same thing).